locales/en/translation.json

```diff
@@ -958,7 +958,8 @@
       "title": "Continue to the service you need to use",
       "header": "Continue to the service you need to use",
       "content": {
-        "paragraph1": "We’ve checked your details. You can now continue to the service you need to use.",
+        "paragraph1": "You’ve successfully proved your identity. You can now continue to the service you want to use.",
+        "paragraph1RepeatFraudCheck": "We’ve checked your details. You can now continue to the service you need to use.",
```

Comment:
Are you sure this is only for repeat fraud checks? Are there any other cases, e.g. update name/address?

Reply:
Thanks Joe, I have created PR to add context for the update name, we already have for the address 
https://github.com/govuk-one-login/ipv-core-back/pull/2780

Reply:
What I mean is, why are we using `repeatFraudCheck` as a context for update name/address journeys - shouldn't we use something like `updateIdentity` or something that captures what the criterion really is?

Reply:
I kinda meant using `updateIdentity` for both - since repeat fraud check is updating your identity (with a fresh fraud check)

I'm happy for you to keep them separate now you've got this, if you prefer.